tests/test_multi_mesh.cpp

```diff
@@ -25,6 +25,30 @@ constexpr PrimitiveType PF = PrimitiveType::Face;
 
 namespace {} // namespace
 
+void print_tuple_map(const DEBUG_TriMesh& parent, const DEBUG_MultiMeshManager& p_mul_manager)
```

Comment:
That looks like debug code. Please add a comment stating that and use `wmtk::logger().debug()` for that.